index.bs

```diff
@@ -940,6 +941,16 @@ spec: webidl
       [[web-background-sync]].
     </p>
   </section>
+  <section>
+    <h3 id="periodic-background-sync">
+      Periodic Background Sync
+    </h3>
+    <p>
+      The <dfn for="PermissionName" enum-value>"periodic-background-sync"</dfn>
+      permission is the permission associated with the usage of
+      [[web-background-sync]].
```

Comment:
If Periodic Background Sync is an API, you should link to it, not to Background Sync. Otherwise, reading this text, it sounds that there are two permissions meant to control the same API.

Reply:
Done. Although I'm not able to directly connect to the new spec even after updating the bibio section using:
bikeshed update --biblio

Anyway, I've added a biblio section up top to link to the PBS spec.